### PR TITLE
Advanced search pos

### PR DESCRIPTION
**Pull request by steven-solar**
Filter by POS and writing intensives in advanced search

### CODE REVIEW

File: static/js/redux/constants/semesterlyPropTypes.jsx [hunk 1]
```diff
@@ -178,6 +178,7 @@ export const schoolSpecificInfo = PropTypes.shape({
   courseRegex: PropTypes.string.isRequired,
   departmentsName: PropTypes.string.isRequired,
   levelsName: PropTypes.string.isRequired,
+  // posName: PropTypes.string.isRequired,
```
Comment (kristinyim) [inline static/js/redux/constants/semesterlyPropTypes.jsx]: uncomment out?

File: static/css/timetable/partials/course_modal.scss [hunk 2]
```diff
@@ -106,6 +115,10 @@
   &.writing {
     background: rgba(55, 89, 148, 0.7);
     border-radius: 7px;
+    margin-top: 0px;
+  }
+  &.search-bar {
+    margin: 0px 0px 0px 3px;
```
Comment (kristinyim) [inline static/css/timetable/partials/course_modal.scss]: just make this: `margin-left: 3px`

File: static/css/timetable/partials/course_modal.scss [hunk 3]
```diff
@@ -173,8 +186,7 @@
     font-family: $font-body;
     font-size: 16px;
     font-weight: 400;
-    margin: 10px 0;
-    padding-right: 15%;
+    margin: 0px 0px 3px 0px;
```
Comment (kristinyim) [inline static/css/timetable/partials/course_modal.scss]: `margin-bottom: 3px`

**Review by kristinyim — CHANGES_REQUESTED** [review]
Nice work! I'd prefer we didn't use negatives in our css so see if you can edit those but its not a big deal if not.

Please fix the other comments I have before merging. I didn't get to run this but lets just merge for now and have people test on staging.

File: static/css/timetable/partials/course_modal.scss (hunk 3 above)
Comment (steven-solar) [reply static/css/timetable/partials/course_modal.scss]: When I change this to just a margin-bottom property, the modal sub-title becomes drastically misaligned (see below). Think it's worth just leaving it like this. 

Screen Shot 2019-06-12 at 8 44 02 PM

**Pull request closed without merging**